### DISCUSSION TITLE
About me

### DISCUSSION THREAD

**Discussion by LivHelen12** [community/community#20213] [Discussions]
Hello,
I'm Livia, a front-end developer from Brazil. 
I have experience in Javascript, Typescript and Reactjs. I'm currently learning about design and React Native.

**Comment by D7EAD** [comment]
Hey, Livia! 

Some great skills you have there! Good luck learning the React Native framework and other UI design concepts!

**Comment by NellyisDevv** [comment]
What's up Livia, I'm trying to get these Javascript skills so I can move on to react! How is React.js treating you compared to Javascript?

**Comment by radar-blip** [comment]
Greetings from USA
Good luck! and keep on learning!!
